.jvmopts

```diff
@@ -1,5 +1,3 @@
 -Dfile.encoding=UTF8
```

Comment:
Maybe this default could be defined explicitly within any test that relies on it.